src/components/ec-modal/ec-modal.story.js

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable no-alert, no-console */
+import { storiesOf } from '@storybook/vue';
+import { boolean } from '@storybook/addon-knobs';
+import EcModal from './ec-modal.vue';
+
+const stories = storiesOf('Modal', module);
+
+stories
+  .add('basic', () => ({
+    components: { EcModal },
+    props: {
+      showModalFromProps: {
+        default: boolean('Show Modal', true),
+      },
+      showHelpLink: {
+        default: boolean('Help Link', true),
+      },
+      showCloseIcon: {
+        default: boolean('Show Close Icon', true),
+      },
+      isLarge: {
+        default: boolean('Is large', false),
+      },
+      negativeHasText: {
+        default: boolean('Negative Button Has Text', true),
+      },
+      positiveHasText: {
+        default: boolean('Positive Has Text', true),
+      },
+    },
+    watch: {
+      showModalFromProps: {
+        immediate: true,
+        handler(newValue) {
+          this.showModal = newValue;
+        },
+      },
+    },
+    data() {
+      return {
+        showModal: false,
+      };
+    },
+    methods: {
+      negativeMethod() {
+        this.showModal = false;
+        console.log('Submit a form or similar');
+      },
+      positiveMethod() {
+        this.showModal = false;
+        console.log('Submit a form or similar');
+      },
+      closeMethod() {
+        console.log('Register the event or similar');
```

Comment:
we can use `actions` addon storybook for showing messages like this without opening the console.